src/routes/chains.rs

```diff
@@ -45,19 +44,13 @@ pub async fn get_chain(context: Context<'_>, chain_id: String) -> ApiResult<cont
  * - `/v1/chains/` Returns the `ChainInfo` for our services supported networks
  *
  */
-#[get("/v1/chains")]
-pub async fn get_chains(context: Context<'_>) -> ApiResult<content::Json<String>> {
-    let mut url = reqwest::Url::parse(base_config_service_url().as_str())
-        .expect("Bad base config service url");
-    url.path_segments_mut()
-        .expect("Cannot add chain_id to path")
-        .extend(["v1", "chains"]);
-
-    Ok(content::Json(
-        RequestCached::new(url.to_string())
-            .request_timeout(chain_info_request_timeout())
-            .cache_duration(chain_info_cache_duration())
-            .execute(context.client(), context.cache())
-            .await?,
-    ))
+#[get("/v1/chains?<limit>")]
+pub async fn get_chains(
+    context: Context<'_>,
+    limit: Option<String>,
```

Comment:
We should probably also take a note in #451 to get rid of `limit` . Generally, like we did for the transaction endpoints, we provide a `page_url` where the clients inject whatever we return in `next` and `previous` in the this struct: 

```pub struct Page<T> {
    pub next: Option<String>,
    pub previous: Option<String>,
    pub results: Vec<T>,
}
```
This would allow us to migrate to cursor based pagination eventually, if we decide on it.

Reply:
I am thinking that it would probably be enough to simply renaming this param to `page_url` and then simply storing `limit=10`  or `limit=10&offset=10` in `next` and `previous` in the response we return